arduino/cores/status.go

```diff
@@ -25,15 +25,11 @@ import (
 )
 
 // Packages represents a set of Packages
-type Packages struct {
-	Packages map[string]*Package // Maps packager name to Package
-}
+type Packages map[string]*Package // Maps packager name to Package
```

Comment:
no reason for this indirection level that also makes the use of the map awkward by having to repeat `packages.Packages`